private/rules/resolver.bzl

```diff
@@ -0,0 +1,5 @@
+def _resolver_impl(repository_ctx):
```

Comment:
What's this for?

Reply:
A vestigial artifact from previous work. Removed.